eslint/package.json

```diff
@@ -12,19 +12,19 @@
   "author": "Alexander Komarov <ak@akxcv.com>",
   "license": "MIT",
   "devDependencies": {
-    "eslint": "^3.19.0",
-    "eslint-plugin-flowtype": "^2.32.1",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jest": "^20.0.1",
-    "eslint-plugin-jsx-a11y": "^5.0.1",
-    "eslint-plugin-node": "^4.2.2",
+    "eslint": "^4.1.1",
+    "eslint-plugin-flowtype": "^2.34.1",
+    "eslint-plugin-import": "^2.6.1",
+    "eslint-plugin-jest": "^20.0.3",
+    "eslint-plugin-jsx-a11y": "^6.0.2",
+    "eslint-plugin-node": "^5.1.0",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-react": "^7.0.1",
-    "yamljs": "^0.2.9"
+    "yamljs": "^0.3.0"
   },
   "peerDependencies": {
-    "eslint": "^3.19.0"
+    "eslint": "^4.1.1"
```

Comment:
shouldn't this dependency be strict? I mean, it should prevent eslint update to 4.2